data/org.freedesktop.impl.portal.Usb.xml

```diff
@@ -68,11 +71,10 @@
       <arg type="o" name="handle" direction="in"/>
       <arg type="s" name="parent_window" direction="in"/>
       <arg type="s" name="app_id" direction="in"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In3" value="QList&lt;std::tuple&lt;QString,QVariantMap,QVariantMap&gt;&gt;"/>
```

Comment:
no because qdbusxml2cpp fails and then I have no idea how to test it (didn't dig). Hence my asking people that know.

(can get the files generated if I had the other annotations back)

Reply:
Can we maybe add a qdbusxml2cpp test to CI?

Reply:
I think qdbusxml2cpp will pass when it has annotations for all types it doesn't understand. 
The other question if those are useful (QList, QPair, basic types in any nesting level) or not. In this case I think it wouldnt work without the user having to write (de)marshalling to get from/to std::tuple<...>

Reply:
so what are the options?

Reply:
> I think qdbusxml2cpp will pass when it has annotations for all types it doesn't understand.

here it returns errors.

Reply:
> Can we maybe add a qdbusxml2cpp test to CI?

Maybe part of #1531 ?